components/base/kinematicbase/differentialDrive.go

```diff
@@ -19,25 +19,6 @@ import (
 	"go.viam.com/rdk/spatialmath"
 )
 
-const (
-	// distThresholdMM is used when the base is moving to a goal. It is considered successful if it is within this radius.
-	distThresholdMM = 3000 // mm
```

Comment:
Looks like you chose to make the default values what we use for moveonMap.  And thats fine, but now these values will be lost.  They will need to be passed into MoveOnGlobe since they are not defaults

Reply:
do you mean just set them in MoveOnGlobe or have these values be parameters to MoveOnGlobe?